plugins/hexbin/filters/HexBin.cpp

```diff
@@ -35,7 +35,7 @@
 #include "HexBin.hpp"
 
 #include <hexer/HexIter.hpp>
-#include <pdal/Geometry.hpp>
+#include <pdal/GEOSUtils.hpp>
```

Comment:
Not sure why this is how it is.  My thought behind Geometry was to hide the implementation, which this undoes.  Not sure if there's some advantage to exposing this.